### ISSUE TITLE
Clarify "Reserve only funds needed..." text in create offer page

### ISSUE THREAD

**Issue by b4n6-b4n6** [haveno-dex/haveno#1294]
Hello!

Currently "create offer" page has
![image](https://github.com/user-attachments/assets/d0eee8aa-e64d-46d6-ba3d-5a24600090fd)
As even technically fluent myself cannot grasp what this I don't see how non-technical people can grasp this

I think can be clarified and made more understandable with change to something like
![image](https://github.com/user-attachments/assets/497cd520-b3f5-463d-bd32-0ad7696d4ce0)
whatever the wording used - important is that that time until offer goes live, balance left in wallet and additional tx fee is clearly displayed for either option

**Comment by shortwavesurfer2009** [comment]
I like the idea as long as anything about UTXOs is stripped. People like you and I understand that term fine, but new people are definitely not going to. That is the same sort of reason that a max button was implemented for sending funds from your wallet to your external wallet because before it was quite confusing on how to do it properly.

**Comment by b4n6-b4n6** [comment]
Okay how about this wording:

---
Funds handling
* Quick (offer will go live instantly and 0.15XMR will be left in your wallet)
* Optimised (offer will go live in ~20 minutes and 1.5XMR will be in your wallet and an additional XMR transaction fee is used)

**Comment by slrslr** [comment]
Current "Reserve only the funds needed" caused confusion in my case. Newbie does not understand what is reserving (which funds) and its impact, that it would cause all available wallet funds being locked and unable to create more offers.

As a result, Funds tab shows zero balances and funds being reserved (for something, it does not suggest to which offer is allocated the balance and on the Funds tab seems to be no link to edit that offer).
![monero-reto-funds-zero-balance](https://github.com/user-attachments/assets/cc86417d-d504-4f85-a337-9b80ea7b3cea)

Ideas:
1) Provide more insight on the Funds tab, for which offer/s is the most funds reserved with a way to go to such offer editing.

2) change tickbox wording:
Reserve only the funds needed and use rest for additional offers. ...
Reserve only necessary amount (amounthere), making rest of the balance (amounthere) available for additional offers.

[ticked by default] Reserve all available balance (amounthere) only for this offer, avoiding mining fee... but requiring new deposit to the wallet in order to create new offer/s.   (if this is too long, maybe an info tooltip icon can be added)

**Comment by boldsuck** [comment]
> Current "Reserve only the funds needed" caused confusion in my case. Newbie does not understand what is reserving (which funds) and its impact, that it would cause all available wallet funds being locked and unable to create more offers.

(which funds)  The exact amount is displayed in the pop-up beforehand and not all available wallet funds are always reserved. If you send 10x 1 XMR to the Haveno wallet, you will have multiple UTXO.

UTXO is not Hveno related. If I have a UTXO of 100 XMR in the Monero GUI and send someone 1 XMR, then the 100 XMR are also unspent. In 20 minutes you have 99 XMR back in change and can spend it.
English is not my native language: but 'reserve only the required amount', is clear and obvious to me.

However, someone is welcome to improve the documentation.
I think that could be added here:
https://boldsuck.github.io/haveno-reto-docs/haveno-ui/haveno-reto_get_started/#31-create-fund-and-publish-your-offer-maker
Or an additional chapter on UTXO management.

**Comment by b4n6-b4n6** [comment]
I recommend to specify the amount of XMR that will be left in the wallet when checkbox is checked

Something like this
![Screenshot from 2024-11-29 19-56-04](https://github.com/user-attachments/assets/05ad6322-7e09-42c6-a746-04aba764d63c)

**Comment by woodser** [comment]
The wording has been updated to:

"Reserve only the necessary funds. Requires a mining fee and ~20 minutes before your offer goes live."

I think this is sufficiently clear.
